addons/l10n_in/models/account_invoice.py

```diff
@@ -541,7 +541,16 @@ def _get_name_invoice_report(self):
         return super()._get_name_invoice_report()
 
     def _post(self, soft=True):
-        """Use journal type to define document type because not miss state in any entry including POS entry"""
+        if any(move.company_id.chart_template == 'in' and move.country_code != 'IN' for move in self):
```

Comment:
This is overkill...  Maybe someone wants to use the India CoA for another country, I don't know.

Reply:
It does not make any sense and if it would, it might be done in generic.

Reply:
I find it already weird that in the settings, you can easily change the fiscal country.  On the other hand, the minute you create an invoice, you are alerted: 
![image](https://github.com/user-attachments/assets/d1b3f1a1-f33f-455a-a02d-f75d669b406a)